packages/documentation/src/pages/getting-started/workflow/overview.mdx

```diff
@@ -8,15 +8,21 @@ VA.gov uses a [Continuous Integration](https://www.thoughtworks.com/continuous-i
 
 1. [Write](/getting-started/workflow/write)
     - Determine which issue to work on
-    - Create feature branch
+    - Create feature branch (naming convention {issue-id}-{subject you are working on})
     - Code new functionality
     - Write tests for new functionality
+    - Cross browser testing (Chrome, IE, Safari, Edge)
 2. [Review](/getting-started/workflow/review)
     - Submit pull request
     - Run unit tests and compliance scans (kicked off when a pull request is created)
+    - Pull request instance will be available for testing (page link should be in the PR)
     - Test and peer review code (read: code review)
+    - Revision request fixes if needed
 3. [Deploy](/getting-started/workflow/deploy)
+    - It is recommended not to merge at the end of the day or right before the weekend unless necessary. 
```

Comment:
Might be good to move this down in the comment below. We should also try to state it in a positive voice explaining what they _should do_ rather than what they _should not_ do.

---

_Each project's code base has a branch called `master` by default. Anything in `master` is deemed to be stable and deployable. This means that **anything you merge will need to be ready to immediately be live or hidden in some way behind a feature flag**._

_Merge changes when support can be provided. Avoid merging significant changes at the end of the day or before the weekend_
```